frontend/packages/data-portal/app/components/AuthorLink/AuthorLink.tsx

```diff
@@ -0,0 +1,79 @@
+import { LinkProps } from '@remix-run/react'
+import { ComponentType } from 'react'
+
+import { EnvelopeIcon } from 'app/components/icons'
+import { Link } from 'app/components/Link'
+import { cns } from 'app/utils/cns'
+
+import { ORC_ID_URL } from './constants'
+import orcIdImage from './orcid.png'
+import { AuthorInfo } from './types'
+
+const BASE_ICON_SIZE = 10
+const LARGE_ICON_SIZE = 14
+
+export function AuthorLink({
+  author,
+  large,
+  LinkComponent = Link,
+}: {
+  author: AuthorInfo
+  large?: boolean
+  LinkComponent?: ComponentType<LinkProps>
+}) {
+  const iconSize = large ? LARGE_ICON_SIZE : BASE_ICON_SIZE
+  const content = (
+    <span className="inline">
+      <span
+        className={cns(
+          'inline border-b',
+
+          author.orcid
+            ? [
+                'border-dashed',
```

Comment:
I'll leave this to @Janeece to confirm but I believe it's supposed to switch from dash to solid on hover

Reply:
yes, dashed to solid on hover